reference/docs-conceptual/components/ise/object-model/The-ISEEditor-Object.md

```diff
@@ -151,7 +162,10 @@ $psISE.CurrentFile.Editor.ToggleOutliningExpansion()
 
 Supported in Windows PowerShell ISE 3.0 and later, and not present in earlier versions.
 
-The read-only Boolean property to indicate whether the caret is next to a parenthesis, bracket, or brace - \(\), \[\], {}. If the caret is immediately before the opening character or immediately after the closing character of a pair, then this property value is **$true**. Otherwise, it is **$false**.
+The read-only Boolean property to indicate whether the caret is next to a parenthesis, bracket, or
+brace - \(\), \[\], {}. If the caret is immediately before the opening character or immediately
```

Comment:
We'd typical put those characters in a code fence instead of escaping them.

Reply:
fixed